src/inputs/BasicInputs/BasicDateInput/BasicDateInput.js

```diff
@@ -56,6 +64,10 @@ BasicDateInput.propTypes = {
    * BasicDateInput's value
    */
   value: PropTypes.object.isRequired,
+  /**
+   * Width in number of columns, from 0 up to 12
+   */
+  nbOfColumns: PropTypes.number,
```

Comment:
Not fan of the prop name :sweat_smile: 
Also, we probably shouldn't make this parameter a configurable prop before we work on the "generic configurable scenario parameters layouts" in the webapp, because we may need different props then, and removing this one will be a breaking change. If we still need this prop in a future version, adding it then won't be a breaking change :slightly_smiling_face:

Reply:
Done.